docs/multitenant.md

```diff
@@ -0,0 +1,7 @@
+### Multitennt agent
+
+Both aca-py agents are exposed via the same Swagger UI under the same url. They are ditinguished under the hood via the authorization method which also specifies a role. Please, rrefer to the workflows doc for further information.
+
+This agent is meant for subwallet and tenant menangement from a tenant perspective.
```

Comment:
TIP: If using VSCode I can recommend the spell checker plugin to prevent typos: https://marketplace.visualstudio.com/items?itemName=streetsidesoftware.code-spell-checker#:~:text=Open%20up%20VS%20Code%20and,and%20reload%20window%20to%20enable.